.github/linters/.yaml-lint.yml

```diff
@@ -0,0 +1,5 @@
+rules:
+  key-duplicates:
+    ignore: |
+      .cirrus.yml
```

Comment:
if you add FreeBSD testing, you'll need this. I can remove from now however if you dont plan on adding.

Reply:
We can keep it in just in case